wled00/net_debug.cpp

```diff
@@ -0,0 +1,75 @@
+#include "wled.h"
+
+#ifdef WLED_DEBUG_NET
+
+NetworkDebugPrinter NetDebug;
+
+void NetworkDebugPrinter::print(const char *s) {
```

Comment:
be a little more defensive, for pointer parameters ensure they are not nullptr otherwise it will crash